index.html

```diff
@@ -220,10 +220,12 @@ <h2>
         </div>
         <form action="https://formspree.io/f/mleznykr" method="POST" class="form">
             <div class="form_list">
-                <input class="form_input form_name form_firs_name" type="text" placeholder="First name" name="user_name"
+                <input class="form_input form_name form_firs_name desktop" type="text" placeholder="First name" name="user_name"
                     required maxlength="30" />
-                <input class="form_input form_name form_last_name" type="text" placeholder="Last name"
+                <input class="form_input form_name form_last_name desktop" type="text" placeholder="Last name"
                     name="user_last_name" required maxlength="30" />
+                <input class="form_input form_name form_full_name mobile" type="text" placeholder="Full name"
+                    name="user_last_name" required maxlength="60" />   
```

Comment:
- [ ] Kindly follow the figma design and add the social media icons below the contact form.
- [ ] Kindly make sure to align the text area with the name and email address by increasing its width.
- Your design:
<img width="407" alt="Screenshot 2022-03-30 at 12 43 14 AM" src="https://user-images.githubusercontent.com/70269507/160712555-0ec002e5-fff5-4116-a4fa-5a4d8f7165ed.png">

- Figma design:
<img width="251" alt="Screenshot 2022-03-30 at 12 43 01 AM" src="https://user-images.githubusercontent.com/70269507/160712589-4e0c9aac-7841-4ec3-81d1-5830059c91dc.png">